### DISCUSSION TITLE
Help with mods

### DISCUSSION THREAD

**Discussion by JoeReeve** [itzg/docker-minecraft-server#2534] [Q&A] [answered]
Hey all, I am having some troubles getting a server up and running and could use some guidance. I can get a vanilla server running just fine but as soon as I add a modpack it breaks. Specifically I'm getting an error that says I am missing a manifest file. I've also tried a couple different modpacks and still this error persists. I feel like I'm overlooking something basic here. I'll attach my yml file and the error log. Any help would be greatly appreciated!
![yml file](https://github.com/itzg/docker-minecraft-server/assets/55531976/2b82b53f-bb10-4766-a7fc-5131775d4535)
![Screenshot from 2023-12-07 10-26-22](https://github.com/itzg/docker-minecraft-server/assets/55531976/c33a9e2a-0145-479a-bbe2-e9e9c8ca5532)

**Comment by itzg** [comment, marked as answer]
I know it sounds counterintuitive, but you'll need to reference the main modpack file and not the server file.

**Reply by JoeReeve** [reply]
Thanks! Really appreciate it:) Now I'm getting some weird  other weird error "cannot access class sun.security.util.ManifestEntryVerifier (in module java.base) because module java.base does not export sun.security.util to unnamed module @0x682a0b20" But progress is progress!

**Reply by JoeReeve** [reply]
Some quick googling leads me to believe it's this mod's version of minecraft having weird issues with the java version on my server

**Reply by itzg** [reply]
Make sure you are using the appropriate image tag for Java version needed

https://docker-minecraft-server.readthedocs.io/en/latest/versions/java/
